flox-bash/lib/commands/development.sh

```diff
@@ -243,8 +246,9 @@ _usage["print-dev-env"]="print shell code that can be sourced by bash to reprodu
 function floxDevelop() {
 	trace "$@"
 	parseNixArgs "$@" && set -- "${_cmdArgs[@]}"
+	parseFloxFlakeArgs "$@" && set -- "${_cmdArgs[@]}"
 
-	local -a developArgs=()
+	local -a developArgs=( "${_floxFlakeArgs[@]}" )
 	local -a installables=()
 	local -a remainingArgs=()
 	while test $# -gt 0; do
```

Comment:
were parsing a subset of nix flake args here again, although refactored into _floxFlakeArgs

Reply:
also in other development commands

Reply:
Thanks for highlighting - pushed a commit to remove the arg parsing now handled by `parseFloxFlakeArgs()`.